### ISSUE TITLE
[BG] Sota,  Vehicle scaling issue

### ISSUE THREAD

**Issue by Pifftannen** [dalaranwow/dalaran-wow#5120]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

DP talent bugged in sota. When passenger in demo it does ridiculous amount of dmg.

**Comment by kysqt** [comment]
Its not just DP, my shaman gets 25k lavabursts on players, the vehicles scale with the players gearscore, and so do the cannons, so if a player with high GS is the driver, the passenger will now scale with the drivers GS too. 

Vehicles should have 80k HP at level 80.

Cannons should have 60k, its a team effort, as it stands now 1 vehicle with high GS driver and 2 rdps can solo the entire enemy team.

Also hunters aspect of the pack works on vehicles.

If you're dead at a graveyard while it gets capped it doesn't teleport you to another graveyard. 

When the boats start moving new players joining on the offensive side get teleported to the shore.

Last time this bugged BG was enabled in the rbg list people stopped queuing bgs, or you would see 10 players join and instantly leave because it was SOTA. Staff please remove it from rbg list until fixed.

**Comment by Aridam** [comment]
The Passenger bug is common, though, critical (the worst one in my opinion) -- it seems that the Ride Vehicle mechanic which scales the Driver's averaged Item Level applies to the boarded Passenger as well, which shouldn't happen. So in other words, everyone who rides alongside, gets that benefit, and as such -- as implied -- two competent RDpS could just carry the entire match for their side.

The provided HP for both Vehicles above me (minus with Ride Vehicle's scaling aura mechanic applied to Demolishers depending on Driver's average IL), is indeed correct.

Also Demolishers are supposed to move _slightly-faster_ than Players (and as such the best approach to keeping up with them is by chasing after them in an intercepting angle, otherwise in a straight line you just fall behind unless mounted or have a Movement Speed increase active at the time), albeit no other Movement Speed effects should affect them -- be it reducing or increasing (that includes the Aspect mention -- it shouldn't work, as nothing should).

Being dead awaiting GY resurrection should get you auto-ported should the GY get capped indeed -- just like all other BGs.

Also, yes, all new people joining during the ships' already-active departure should _still_ be boarded on them rather than spawn them on the piers already.

Due to how bugged the Battleground itself is (especially with the Passenger one still existing), it'd make for an unpleasant time for most people attending it, as it all makes it more of a "who abuses more wins" type than anything -- logically, it should be temporarily-closed until at the very least the Passenger bug has been fixed.

**Comment by Zarjofgt** [comment]
Honestly its probably only open so people can get their chef achievement completed.

**Comment by Aridam** [comment]
Still a waste of time for many other, uninterested people, either way. Should be checked.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the reports, I will work on this during this week, been quite busy with #561.

I did port from Rushor's previous commits or fix myself every reported bug from The Strand of Ancients before its reopening and since I did not know about the mentioned issues here until now, they went live with it.

Nonetheless, I will take a look during the next few days.

**Comment by Lothloryen** [comment]
Hello there,

Fixed the issue with the passengers of vehicles (not only from the Strand of Ancient, it is general) being able to benefit from the gear scalling spell, it will be on the live realm with the next maintenance.

**Comment by Lothloryen** [comment]
Hello again,

The issue where players waiting to get revived on a graveyard were not being relocated when such graveyard is capped by the opposite faction is now corrected. I will apply it to Algalon together with the previously mentioned fix tomorrow.

**Comment by Lothloryen** [comment]
Also fixed the issue with demolishers being able to benefit from the Hunter's Aspect of the Pack.

**Comment by Lothloryen** [comment]
Hello again, the mentioned fixes are already on Algalon.

**Comment by Lothloryen** [comment]
Hello again,

Vehicles should have the correct HP already (before scale with the player's gear). The issue about players being teleported directly when logging while the ship is moving is related to a general issue with transports wich will be reviewed a part from this.

So, closing this issue. If you find anything more, any kind of feedback will be more than welcome.

Thank you everyone.
